pkg/arvo/sur/chat-push-hook.hoon

```diff
@@ -0,0 +1,24 @@
+^?
+|%
++$  action
+  $%  ::  %add: make a local chatroom accessible to foreign ships
+      ::
+      [%add =path allow-history=?]
```

Comment:
Same, this should be moved to a resource identifier, not a path.

Reply:
Good point.